web-extension/search.js

```diff
@@ -112,6 +112,7 @@ function _displaySearchResults(response, isHostQuery) {
                 _onEntryAction
             )
         );
+        entry.appendChild(createButtonWithCallback('open', result, null, event => _onEntryOpen(event.target)));
```

Comment:
Can we give these buttons a simple tooltip? Maybe this can be a separate improvement.

Reply:
Good idea!

Reply:
-> #73